CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
+0.35.0 / TBA
```

Comment:
No need for a new version (the previous one hasn't been released yet), just add the change to `0.34.0`